docs/contributing/governance/veyfi.md

```diff
@@ -6,7 +6,7 @@ Contract Address: [`0x90c1f9220d90d3966fbee24045edd73e1d588ad5`](https://ethersc
 
 ## Specification
 
-Governance Forum Thread: https://gov.yearn.fi/t/yip-65-evolving-yfi-tokenomics
+Governance Forum Thread: <https://gov.yearn.fi/t/yip-65-evolving-yfi-tokenomics>
```

Comment:
can you please remove the <> around links? we don't have this standard in the other pages of the docs!